components/modules/activity/ActivityHistoryEntry.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Text, View } from 'react-native';
+
+import CustomImage from '@/components/ui/image/CustomImage';
+
+interface ActivityHistoryEntryProps {
+  activityAction: 'created' | 'updated' | 'deleted' | 'accepted' | 'rejected';
+  activityType: 'expense' | 'payment';
+  position: 'left' | 'right';
+  imageUri: string;
+  username: string;
+  date: Date;
+  message?: string;
+}
+
+const ActivityHistoryEntry: React.FC<ActivityHistoryEntryProps> = ({
+  activityAction,
+  activityType,
+  position,
+  imageUri,
+  username,
+  date,
+  message,
+}) => {
+  const { t } = useTranslation();
+
+  const flexRow = position === 'left' ? 'flex-row' : 'flex-row-reverse';
```

Comment:
could we merge that two one if for example flex parameters or sth else?

Reply:
which two?